misc/upload-pypi.py

```diff
@@ -124,7 +124,9 @@ def make_sdist(self) -> None:
 
     def download_compiled_wheels(self) -> None:
         self.heading('Downloading wheels compiled with mypyc')
-        self.run_in_virtualenv('misc/download-mypyc-wheels.py %s' % self.version)
+        self.run_in_virtualenv(
+            '%s %s' %
+            (os.path.abspath('misc/download-mypyc-wheels.py'), self.version))
```

Comment:
This makes it so that we run the download-mypyc-wheels script in the current directory and not in the checkout, since I fixed the script after. I'll add a comment